internal/clients/authentication.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 type Authentication struct {
+	ResourceID   string              `json:"resource_id"`
```

Comment:
Could we name it ApplicationID or even SourcesApplictionID?
If you don't agree, I'd at least document it, ResourceID is very general, which is intentional in Sources, where it can be ID of "anything", but for us it should always be ApplicationID, right?

Reply:
sure, ill change the name :)

Reply:
What about the ResourceID in SourcesInfo, is that OK or do you think I should change that as well?

Reply:
Ideally I'd change that as well, but that one lives just in that single file, so that one is not as important to change, but sure, once we are at it :+1:
In the result, it should be ResourceID tho, as that's what we send to sources.